### DISCUSSION TITLE
Welcome to zi-dataset Discussions!

### DISCUSSION THREAD

**Discussion by secsilm** [secsilm/zi-dataset#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mumchristmas** [comment]
Alan 您好，想请教一下这个数据集中叶子部件等汉字结构分析的工具或数据源：）

**Reply by secsilm** [reply]
你好，是我从 [MDBG Chinese Dictionary](https://www.mdbg.net/chinese/dictionary) 抓取下来的。

**Reply by mumchristmas** [reply]
感谢！
